src/algebra/lie/basic.lean

```diff
@@ -148,13 +159,13 @@ identity. Forgetting the scalar multiplication, every Lie algebra is a Lie ring.
 class lie_algebra (R : Type u) (L : Type v) [comm_ring R] [lie_ring L] extends semimodule R L :=
 (lie_smul : ∀ (t : R) (x y : L), ⁅x, t • y⁆ = t • ⁅x, y⁆)
 
-@[simp] lemma lie_smul  (R : Type u) (L : Type v) [comm_ring R] [lie_ring L] [lie_algebra R L]
+lemma lie_smul (R : Type u) (L : Type v) [comm_ring R] [lie_ring L] [lie_algebra R L]
```

Comment:
Maybe add a comment explaining why these aren't simp? (Future me would probably try to make them simp if I was browsing through this file.)

Reply:
These lemmas no longer exist so the problem is gone!